README.md

```diff
@@ -256,11 +256,6 @@ A complete API definition is in the works. Meanwhile, you can learn how to you u
   - [`ipfs.block.put(block, cid, [callback])`](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/BLOCK.md#put)
   - [`ipfs.block.stat(cid, [callback])`](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/BLOCK.md#stat)
 
-- [repo](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/)
-  - `ipfs.repo.init`
-  - `ipfs.repo.version`
-  - `ipfs.repo.gc` (not implemented, yet!)
```

Comment:
Why move these to the Graph section??

Reply:
I would agree if moved to the Node Management Section

Reply:
It's on the Node Management section

Reply:
<img width="1397" alt="image" src="https://user-images.githubusercontent.com/1211152/36250595-98fe4106-123e-11e8-916f-f34103a5f251.png">

Reply:
nvm, I got fooled twice by GH interface. all good :)